system/t06_publish/PublishUpdate7Test_gold

```diff
@@ -1,8 +1,6 @@
 Loading packages...
 Generating metadata files and linking package files...
 Finalizing metadata files...
-Signing file 'Release' with gpg, please enter your passphrase when prompted:
```

Comment:
why is signing gone ?

Reply:
Signing is not gone, refer to to `PublishUpdate6Test_gold`.

Reply:
ah, the renumbering... I would maybe undo that, makes it hard to review and compare to older runs... what do you think ?